common/app/common/dfp/DfpData.scala

```diff
@@ -457,7 +456,7 @@ case class GuCreativeTemplate(id: Long,
                               snippet: String,
                               creatives: Seq[GuCreative]) {
 
-  val examplePreviewUrl: Option[String] = creatives.headOption flatMap { creative => creative.previewUrl }
+  lazy val examplePreviewUrl: Option[String] = creatives flatMap {_.previewUrl} headOption
```

Comment:
Yeah - using `find` would be quicker but it looked a bit clunky and I went for aesthetics!